frontend/src/components/University/TopUniCard.jsx

```
@@ -0,0 +1,36 @@
+  const navigate = useNavigate();
+
+  return (
+    <div className="p-4 shadow-md bg-white rounded-2xl">
+      <h3 className="text-lg font-bold mb-4 text-gray-800">Top Universities</h3>
+      
+      {topUniversities.map((uni, index) => (
+        <div
+          key={uni.id}
+          className="mb-3 cursor-pointer hover:bg-gray-50 p-2 rounded-lg transition"
+          onClick={() => navigate(`/university/${uni.name}`)}
+        >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)